### ISSUE TITLE
Ctrl-Alt simultaneously does not work (Windows)

### ISSUE THREAD

**Issue by kpym** [adobe/brackets#12306]
For shortcuts that starts with <kbd>Ctrl-Alt</kbd> if I press <kbd>Ctrl</kbd> and then <kbd>Alt</kbd>, or <kbd>Alt</kbd> and then <kbd>Ctrl</kbd>, it works ok. But if I press <kbd>Ctrl</kbd> and <kbd>Alt</kbd> simultaneously it doesn't.

I'm on Windows 7 (32bits) with Brackets Release 1.6 build 1.6.0-16680 (release 3df0ac6fa).

**Comment by abose** [comment]
Can't repro this. How would you press `control+alt` simultaneously?
Does this happen for all shortcuts or just some specific shortcut, can u give specific examples?

**Comment by kpym** [comment]
I agree it is strange. But yes when I press both in the same time ALL shortcuts starting with <kbd>Ctrl-Alt</kbd> are not working. It is possible that this is related to my keyboard, but I don't have such problem with other programs (Sublime for example).

**Comment by kpym** [comment]
I checked my key codes [here](https://www.w3.org/2002/09/tests/keys.html), and there is no difference between "simultaneously" or not. In any case it sends first one and then the other. But with brackets for me there is a difference ... very strange.
